### PR TITLE
ERP-3774 Show overdue amount in ledger

### PR DESCRIPTION
**Pull request by ankur-singh-dehaat**

### CODE REVIEW

**Comment by shivam21** [comment]
@ankur-singh-dehaat please correct commit id in commit

File: lib-ledger/src/main/java/lib/dehaat/ledger/presentation/mapper/LedgerViewDataMapper.kt [hunk 1]
```diff
@@ -394,4 +408,8 @@ class LedgerViewDataMapper @Inject constructor() {
             interestDays = it.interestDays
         )
     }
+
+    private fun Double.isPositive() = this > 0.0
```
Comment (anil-agridroid) [inline lib-ledger/src/main/java/lib/dehaat/ledger/presentation/mapper/LedgerViewDataMapper.kt:412]: Change fun name to isGreaterThanZero

File: lib-ledger/src/main/java/lib/dehaat/ledger/presentation/mapper/LedgerViewDataMapper.kt [hunk 2]
```diff
@@ -394,4 +408,8 @@ class LedgerViewDataMapper @Inject constructor() {
...
+
+    private fun Double.isNegative() = this < 0.0
```
Comment (anil-agridroid) [inline lib-ledger/src/main/java/lib/dehaat/ledger/presentation/mapper/LedgerViewDataMapper.kt:414]: Use fun name isSmallerThanZero

**Review by anil-agridroid — CHANGES_REQUESTED** [review]

File: lib-ledger/src/main/java/lib/dehaat/ledger/presentation/mapper/LedgerViewDataMapper.kt (hunk 2 above)
Comment (ankur-singh-dehaat) [reply lib-ledger/src/main/java/lib/dehaat/ledger/presentation/mapper/LedgerViewDataMapper.kt]: addressed, can you please review this again?

**Pull request merged**